internal/exec/stages/disks/disks.go

```diff
@@ -109,15 +136,17 @@ func (s stage) createPartitions(config types.Config) error {
 		devs = append(devs, string(disk.Device))
 	}
 
-	if err := s.waitOnDevices(devs, "disks"); err != nil {
+	if err := s.waitOnDevicesAndCreateAliases(devs, "disks"); err != nil {
 		return err
 	}
 
 	for _, dev := range config.Storage.Disks {
+		devAlias := eutil.DeviceAlias(string(dev.Device))
```

Comment:
You thought wrong:
gopath/src/github.com/coreos/ignition/internal/exec/stages/disks/disks.go:144: cannot use dev.Device (type types.Path) as type string in argument to "github.com/coreos/ignition/internal/exec/util".DeviceAlias